pyspecdata/nnls.py

```diff
@@ -2,7 +2,7 @@
 
 from .general_functions import redim_F_to_C, redim_C_to_F, strm, inside_sphinx
 if not inside_sphinx():
-    from . import _nnls
+    from . import nnls
```

Comment:
this does not seem right.

Reply:
if you include the underscore there is a circular looping error. This fixes that error

Reply:
give an example of how I would reproduce this error, please

Reply:
Meaning: I changed it back now.  I can run the example `fit_fake_data.py` and it runs fine.  Can you give me an example where  you get the circular import error you describe?